docs/source/profiler.rst

```diff
@@ -761,6 +761,13 @@ Below is an breakdown of all the options.
       * top_k_categories - (int) Number of categories to be displayed when reporting
       * max_sample_size_to_check_stop_condition - (int) The maximum sample size before categorical stop conditions are checked
       * stop_condition_unique_value_ratio - (float) The highest ratio of unique values to dataset size that is to be considered a categorical type
+      * cms - (Boolean) Enables or Disables the use of count min sketch / heavy hitters for approximate frequency counts.
```

Comment:
oh interesting... I was expecting `CMS` to be its own options class .... 🤔

Reply:
yup it's a method for doing approximate frequency counts. Only intended for the categorical profile for now.